website/tutorial/basic/00-introduction.mdx

```diff
@@ -0,0 +1,56 @@
+---
+id: 00-introduction
+title: Introduction
+sidebar_label: Introduction
+---
+
+### Overview
+
+GraphQL is the rising star of backend technologies. It's a popular alternative for REST as an API design paradigm and is becoming the new
+standard for exposing the data and functionality of a web server.
+
+In this tutorial, you'll learn how to build an _idiomatic_ GraphQL server entirely from scratch.
+
+You are going to use the following technologies:
+
+- [Node.js](https://nodejs.org/) - as an engine and runtime for our server.
+- [TypeScript](https://www.typescriptlang.org/) - a strongly typed programming language which builds on JavaScript giving you better tooling at any scale.
+- `graphql-yoga`- the easiest way of running a GraphQL server.
+- [`graphql-js`](https://github.com/graphql/graphql-js) - we will use the core `graphql` library as execution engine for our server.
+- [GraphiQL](https://github.com/graphql/graphiql): A graphical interface that allows you to interactively
+  explore the functionality of a GraphQL API by sending query, mutation and subscription operations. It's somewhat similar to
+  [Postman](https://www.getpostman.com/) which offers comparable functionality for REST APIs. Among other things,
+  GraphiQL:
+  - Auto-generates comprehensive documentation for all available API operations.
+  - Provides an editor where you can write queries, mutations & subscriptions, with auto-completion(!) and syntax
```

Comment:
What do you mean by `with auto-completion(!)`?

Maybe: `with auto-completion / explorer`